drivers/timer/realtek_rts5912_rtmr.c

```diff
@@ -0,0 +1,242 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+ * Author: Lin Yu-Cheng <lin_yu_cheng@realtek.com>
+ */
+
+#define DT_DRV_COMPAT realtek_rts5912_rtmr
+
+#include <zephyr/init.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/sys_clock.h>
+#include <zephyr/irq.h>
+#include <zephyr/spinlock.h>
+#include <zephyr/drivers/timer/system_timer.h>
+
+#include <reg/reg_rtmr.h>
+#include <reg/reg_system.h>
+
+#define RTS5912_SCCON_REG_BASE ((SYSTEM_Type *)(DT_REG_ADDR(DT_NODELABEL(sccon))))
+
+#define CYCLES_PER_TICK (CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC / CONFIG_SYS_CLOCK_TICKS_PER_SEC)
+
+#define RTMR_REG ((RTOSTMR_Type *)DT_INST_REG_ADDR(0))
+
+#define SLWTMR_REG ((RTOSTMR_Type *)(DT_REG_ADDR(DT_NODELABEL(slwtmr0))))
```

Comment:
Instead of using a nodelabel here, I recommend using the `DT_COMPAT_GET_ANY_STATUS_OKAY`
```
DT_REG_ADDR(DT_COMPAT_GET_ANY_STATUS_OKAY(realtek_rts5912_slwtmr0))
```

Reply:
Thanks for your recomment, we will change to this way.